lib/test_case.dart

```diff
@@ -129,6 +129,23 @@ class TestCase {
     return TestCase.fromJson(response);
   }
 
+  static Future<SoftDeleteTestCase> delete(
+    int caseId, {
+    bool soft = false,
+  }) async {
+    final parametersMap = {
+      'soft': (soft == true) ? 1 : 0,
```

Comment:
`soft` isn't nullable
```suggestion
      'soft': soft ? 1 : 0,
```